### ISSUE TITLE
build task validate:rule does not stop the build on error

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#2591]
Apparently it reports the problem but doesn't prevent the build from completing, which doesn't help us catch any validation issues. In this case the new `aria-command-name` rule doesn't have a `cat.` tag.

![image](https://user-images.githubusercontent.com/2433219/97022898-f55be600-1511-11eb-96ef-3b04417bf43e.png)

**Comment by padmavemulapati** [comment]
Verified in the latest build: Not seeing `failure but skipped build validation` task. Also seeing the category tag in the rule json object

![image](https://user-images.githubusercontent.com/47409579/97829680-b9363b80-1cf0-11eb-88d6-961873d6b1ec.png)

![image](https://user-images.githubusercontent.com/47409579/97829713-d8cd6400-1cf0-11eb-8745-ed3b1e1853ae.png)
